src/components/sticky/sticky.js

```diff
@@ -169,22 +171,22 @@ function MdSticky($document, $mdConstant, $compile, $$rAF, $mdUtil) {
         setCurrentItem(null);
 
       // Going to next item?
-      } else if (isScrollingDown && self.next) {
-        if (self.next.top - scrollTop <= 0) {
+      } else if (isScrollingDown) {
+        if (self.next && self.next.top - scrollTop <= 0) {
```

Comment:
I know it was like this before, but could you change this? 
`self.next.top - scrollTop <= 0` is a less readable way of saying `self.next.top <= scrollTop`